pages/nft-book-store/collection/new.vue

```diff
@@ -384,6 +517,10 @@ const notificationEmailInput = ref('')
 const isStripeConnectChecked = ref(false)
 const stripeConnectWallet = ref('')
 const stripeConnectWalletInput = ref('')
+const mustClaimToView = ref(true)
+const hideDownload = ref(false)
```

Comment:
collection does not have `mustClaimToView` and `hideDownload` since they are inside setting of individual books

Reply:
[🔥 Remove DRM setting in collection](https://github.com/likecoin/nft-book-press/pull/134/commits/7595d3d8e653e7784ab790b13362789bd15f1804)